advanced/is-pattern-1/templates/identity-server-ingress.yaml

```diff
@@ -19,7 +19,8 @@ metadata:
   namespace : {{ .Release.Namespace }}
   annotations:
     kubernetes.io/ingress.class: "nginx"
-    nginx.ingress.kubernetes.io/ssl-passthrough: "true"
+    nginx.ingress.kubernetes.io/backend-protocol: "HTTPS"
+    #nginx.ingress.kubernetes.io/ssl-passthrough: "true"
```

Comment:
@Chathuru no need to maintain this. Remove it.

Reply:
Remove in https://github.com/Chathuru/kubernetes-is/commit/f0001c25d74c361eb8fcd41ca383bfbefb481822